src/main/java/com/robohorse/robopojogenerator/generator/postprocessors/AbsPostProcessor.java

```diff
@@ -10,69 +10,87 @@
 import com.robohorse.robopojogenerator.models.GenerationModel;
 
 import javax.inject.Inject;
+
 import java.util.Set;
 
 /**
  * Created by vadim on 23.10.16.
  */
 public abstract class AbsPostProcessor {
+
     @Inject
-    ClassGenerateHelper generateHelper;
+    ClassGenerateHelper    generateHelper;
     @Inject
     ClassTemplateProcessor classTemplateProcessor;
 
+
     public String proceed(ClassItem classItem, GenerationModel generationModel) {
+
         applyAnnotations(generationModel.getAnnotationItem(), classItem);
         return proceedClass(classItem, generationModel);
     }
 
+
     public abstract String proceedClassBody(ClassItem classItem, GenerationModel generationModel);
 
     public abstract String createClassTemplate(ClassItem classItem, String classBody);
 
+
     private String proceedClass(ClassItem classItem, GenerationModel generationModel) {
-        final String classBody = proceedClassBody(classItem, generationModel);
-        final String classTemplate = createClassTemplate(classItem, classBody);
-        final Set<String> imports = classItem.getClassImports();
+
+        final String        classBody      = proceedClassBody(classItem, generationModel);
+        final String        classTemplate  = createClassTemplate(classItem, classBody);
+        final Set<String>   imports        = classItem.getClassImports();
         final StringBuilder importsBuilder = new StringBuilder();
         for (String importItem : imports) {
             importsBuilder.append(importItem);
             importsBuilder.append(ClassTemplate.NEW_LINE);
         }
+
+        return createClassItemText(classItem.getPackagePath(),
+                                   importsBuilder.toString(),
+                                   classTemplate);
+    }
+
+
+    protected String createClassItemText(String packagePath, String imports, String classTemplate) {
```

Comment:
I make the `classTemplateProcessor.createClassItem()` into this method.

It is convenient to override and remove the semicolon in my post processor.